rln/src/ffi.rs

```diff
@@ -212,7 +332,38 @@ mod test {
         let result_data = <&[u8]>::from(&output_buffer).to_vec();
         let (root_batch, _) = bytes_le_to_field(&result_data);
 
+        // We check if roots are the same
         assert_eq!(root_single, root_batch);
+
+        // We now delete all leaves set and check if the root corresponds to the empty tree root
+        // delete calls over indexes higher than no_of_leaves are ignored and will not increase self.tree.next_index
+        for i in 0..2 * no_of_leaves {
```

Comment:
This line is a bit confusing. Can we pull out the range into a separate variable perhaps? Not do calculation (* no_of_leaves) in range itself

Reply:
Sure! Addressed in https://github.com/vacp2p/zerokit/pull/22/commits/28b96a60a631beaebf8386ac5f727322c759d440